webui/react/src/components/kit/Icon.tsx

```diff
@@ -117,11 +204,28 @@ const Icon: React.FC<Props> = ({
 }: Props) => {
   const classes = [css.base];
 
+  const rowIcon = React.useMemo(() => {
+    if (size === 'small') return <RowIconSmall />;
```

Comment:
`size` controls the size of the icon. These are fully different icons. I also notice that you're not handling the size correctly on any of the other icons (they're always 24x24).